Gemfile

```diff
@@ -7,7 +7,7 @@ rails_version = "#{ENV['RAILS_VERSION'] || '6.0.3.3'}"
 
 gem "rake", "~> 12.0"
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem "rails", rails_version == "main" ? { github: "rails/rails" } : rails_version
```

Comment:
I think `bundler` is defaulting to `master` instead of the default branch so we have to be explicit here. Changing to `git` and the full https path uses the secure approach so we can do away with the warning we were seeing.